### DISCUSSION TITLE
will it be possible to send a shortcut command to my pc

### DISCUSSION THREAD

**Discussion by benimars23** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#575] [Q&A]
im looking for a way to adjust a led light i control through software on my pc through google assist routines. will it be possible to do this ?

i already have a shortcut command on my pc to turn on the light but i was hoping to automate it through google assist so i can have it turned on every night at sunset.

**Comment by im-coder-lg** [comment]
No, it won't happen since GAUDC needs to be deeply integrated, which is hard to do.
